### PR TITLE
fix: enforcer is null

### PR DESCRIPTION
**Pull request by arteevraina**
Signed-off-by: Arteev Raina <arteevraina@gmail.com>

### Description :

Explicitly handle the error for enforcer being null.

### Please ensure Checkmark the following :

* [x] - Tested Localy
* [x] - Checked in IE11 and Firefox
* [ ] - Checked in mobile
* [ ] - Backend work
* [x] - Frontend work
* [x] - Bug Fixes
* [ ] - Enhancements

### CODE REVIEW

**Comment by arteevraina** [comment]
@hsluoyz This PR will show an error until we merge #103.

File: web/src/EnforcerPage.js [hunk 1]
```diff
@@ -36,16 +36,18 @@ class EnforcerPage extends React.Component {
...
-        enforcer: enforcer,
-        adapters: adapters,
-        models: models,
-        adapter: adapters.filter(adapter => adapter.id === enforcer.adapter)[0],
-        model: models.filter(model => model.id === enforcer.model)[0],
-      });
-
-      this.getAdapterPolicies(enforcer.adapter);
-      this.getAdapterGroupingPolicies(enforcer.adapter);
+      if(enforcer != null) {
```
Comment (nodece) [inline web/src/EnforcerPage.js:39]: we don't should lost `undefined`.

Comment (arteevraina) [reply web/src/EnforcerPage.js:39]: @nodece for both `undefined` and `null` do we need not run this code?

Comment (nodece) [reply web/src/EnforcerPage.js:39]: maybe we should add error message to the component. what do you think?

Comment (nodece) [reply web/src/EnforcerPage.js:39]: @arteevraina

Comment (arteevraina) [reply web/src/EnforcerPage.js:39]: both `undefined `and `null` will mean falsy value therefore we won't get the error. If we do something like this
```

if(enforcer) {
// do something
}
```

Please correct me if I am wrong.

**Review by nodece — CHANGES_REQUESTED** [review]

File: web/src/EnforcerPage.js (hunk 1 above)
Comment (arteevraina) [reply web/src/EnforcerPage.js:39]: sure thing!

**Comment by arteevraina** [comment]
@nodece Made the required changes :)

**Comment by hsluoyz** [comment]
@nodece please review.

**Pull request merged**